lib/streamlit/app_session.py

```diff
@@ -43,41 +46,41 @@
     from streamlit.state.session_state import SessionState
 
 
-class ReportSessionState(Enum):
-    REPORT_NOT_RUNNING = "REPORT_NOT_RUNNING"
-    REPORT_IS_RUNNING = "REPORT_IS_RUNNING"
+class AppSessionState(Enum):
+    APP_NOT_RUNNING = "APP_NOT_RUNNING"
+    APP_IS_RUNNING = "APP_IS_RUNNING"
     SHUTDOWN_REQUESTED = "SHUTDOWN_REQUESTED"
 
 
-class ReportSession(object):
+class AppSession:
     """
-    Contains session data for a single "user" of an active report
+    Contains session data for a single "user" of an active app
     (that is, a connected browser tab).
 
-    Each ReportSession has its own Report, root DeltaGenerator, ScriptRunner,
+    Each AppSession has its own SessionData, root DeltaGenerator, ScriptRunner,
     and widget state.
 
-    A ReportSession is attached to each thread involved in running its Report.
+    An AppSession is attached to each thread involved in running its script.
 
     """
 
     def __init__(
         self,
-        ioloop,
-        script_path,
-        command_line,
-        uploaded_file_manager,
-        message_enqueued_callback,
+        ioloop: tornado.ioloop.IOLoop,
+        session_data: SessionData,
+        uploaded_file_manager: UploadedFileManager,
+        message_enqueued_callback: Optional[Callable[[], None]],
+        local_sources_watcher: LocalSourcesWatcher,
     ):
-        """Initialize the ReportSession.
+        """Initialize the AppSession.
 
         Parameters
         ----------
         ioloop : tornado.ioloop.IOLoop
             The Tornado IOLoop that we're running within.
 
         script_path : str
-            Path of the Python file from which this report is generated.
+            Path of the Python file from which this app is generated.
```

Comment:
nit: this docstring parameter is now stale with `session_data` replacing `script_path`